src/Views/Content.elm

```diff
@@ -73,7 +73,7 @@ view model =
     case model.currentPage of
         Home ->
             div [ class "section--vertical-fill-center" ]
-                [ section [ class "section section--highlight section--align-bottom" ]
+                [ section [ class "section section--lighter section--align-bottom" ]
```

Comment:
I guess it doesn't matter these are no longer in sync with Haven if we do this - since that site and these have diverged a fair bit now.

Reply:
They could do with better names that don't tie them to a particular tone of lighter or darker.